scss/core/_variables.scss

```diff
@@ -121,7 +121,7 @@ $container-max-widths: (
   sm: 540px,
   md: 720px,
   lg: 960px,
-  xl: 1140px
+  xl: 1140px + 2 * $grid-gutter-width,
```

Comment:
This hasn't been defined yet, best to keep this change out of `core/*` and bring it into `edx/*`